packages/odyssey-react/src/components/Heading/Heading.tsx

```diff
@@ -57,23 +70,27 @@ let Heading = forwardRef<HTMLHeadingElement, HeadingProps>((props, ref) => {
     children,
     noEndMargin = false,
     lineHeight,
+    color = "heading",
+    weight = "bold",
     ...rest
   } = props;
-
   const Tag = `h${level}` as const;
-
+  const size = visualLevel
+    ? (`heading${visualLevel}` as const)
+    : (`heading${level}` as const);
   const componentClass = useCx(
     styles.root,
     visualLevel && styles[`level${visualLevel}`],
-    noEndMargin && styles.noEndMargin,
-    lineHeight && styles[`${lineHeight}LineHeight`]
+    noEndMargin && styles.noEndMargin
   );
-
+  const lh = lineHeight ? lineHeight : parseInt(level) > 3 ? "title" : "base";
   const omitProps = useOmit(rest);
 
   return (
     <Tag {...omitProps} ref={ref} className={componentClass}>
```

Comment:
With the other additions to Text, I'm wondering if we should expand the tags as well to accept `h1` - `h6`

Reply:
☝🏼 this fits with my understanding of `Text` as "text element inserter" rather than "leaf styler".

Currently, it does both. IMO our utility should have a more clear use case / both uses should not be valid as they produce inconsistent code.

Reply:
The benefit here would be one less intermediate element in the DOM to render a `<Heading />` yes? Drawbacks would be more surface area in `<Text />`, anything else?

Reply:
Proposal based on this morning's convo: https://c198ad7.ods.dev/?path=/story/utilities-utilities--page#heading